networkx/algorithms/broadcasting.py

```diff
@@ -129,9 +138,9 @@ def tree_broadcast_time(G, node=None):
 
     References
     ----------
-    .. [1] Harutyunyan, H. A. and Li, Z. "A Simple Construction of Broadcast Graphs."
-    In Computing and Combinatorics. COCOON 2019 (Ed. D. Z. Du and C. Tian.) Cham,
-    Switzerland: Springer, pp. 240-253, 2019.
+    .. [1] Harutyunyan, H. A. and Li, Z. "A Simple Construction of
+    Broadcast Graphs." In Computing and Combinatorics. COCOON 2019
+    (Ed. D. Z. Du and C. Tian.) Cham, Switzerland: Springer, pp. 240-253, 2019.
```

Comment:
Lets make the line break between the list of authors and the title. Then probably another after the title. Something like:
```suggestion
    .. [1] Harutyunyan, H. A. and Li, Z.
        "A Simple Construction of Broadcast Graphs."
        In Computing and Combinatorics. COCOON 2019
        (Ed. D. Z. Du and C. Tian.) Springer, pp. 240-253, 2019.
```